portfolio/src/main/webapp/index.html

```diff
@@ -10,9 +11,19 @@
     <script src="script.js"></script>
   </head>
   <body>
+    <nav class="navbar navbar-light bg-primary navbar-extend-lg">
+        <ul class="nav navbar-nav navbar-left">
+          <a class="nav-text navbar-brand" href="#"><span class="text-emphasis">Antonio</span>'s Portfolio</a>
```

Comment:
nit: This should probably be nested in a `<li>` since it's within a `<ul>`

Reply:
Fixed!